apps/web/src/views/universalFarms/components/PoolsFilterPanel.tsx

```diff
@@ -11,20 +11,25 @@ import styled from 'styled-components'
 import { CHAINS } from 'config/chains'
 import { ASSET_CDN } from 'config/constants/endpoints'
 import { useAllTokensByChainIds } from 'hooks/Tokens'
-import { useMemo } from 'react'
+import React, { useMemo } from 'react'
 import { useTranslation } from '@pancakeswap/localization'
 import { getChainNameInKebabCase } from '@pancakeswap/chains'
 import { Protocol } from '@pancakeswap/farms'
 import { ERC20Token } from '@pancakeswap/sdk'
+import { getChainFullName } from '../utils'
 
-const PoolsFilterContainer = styled(Flex)`
+const PoolsFilterContainer = styled(Flex)<{ $childrenCount: number }>`
   flex-wrap: wrap;
   justify-content: flex-start;
   gap: 16px;
 
   & > div {
     flex: 1;
-    max-width: calc(33% - 16px);
+  }
+
+  & > div:nth-child(1),
+  & > div:nth-child(2) {
+    width: calc(${({ $childrenCount: $childCount }) => `${100 / $childCount}%`} - 16px);
```

Comment:
maybe use grid is better?

Reply:
yeah, note it and optimize in the future after other issues fixed